templates/fullstack/server/src/schema/generated.graphql

```diff
@@ -1,53 +0,0 @@
-
-
```

Comment:
What was the motivation to remove this file and one in client?

Reply:
In general, if I get the package right, the idea is to guide new users into graphQL.

So what I was thinking is that just after the init:
1) we suggest to `git init`. => So that people have a clean starting point
2) we ask to follow README.md => So that we don't write 2 times "almost" the same things... But not exactly 😆 

Then, following the README, cmd after cmd, people will see changes in their repo.
Like this, they will get a direct view of what cmd is doing what action.

It's only my viewpoint, feel free to comment

Reply:
Really nice point of view. I we would follow that we will probably need to remove other generated content from server that will leave only index file there. We also have client only template with no chamges